firefox-ios/Client/Frontend/Home/LegacyHomepageViewController.swift

```diff
@@ -162,6 +161,8 @@ class LegacyHomepageViewController:
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
+        // FXIOS-9428 - Record telemetry in viewDidAppear since viewWillAppear is sometimes triggered twice
+        viewModel.recordViewAppeared()
```

Comment:
I was hoping to get @cyndichin 's eyes on this since she originally investigated it but looks like she is out until 12/16.

I don't recall offhand why this approach wasn't used originally (rather than the throttling). I'm assuming it was because we were seeing duplicate `viewDidAppear` calls also, is that not happening for you in practice? (I actually thought that as a general rule the did-appear calls were always paired with will-appear, but I could be forgetting my basics here).

Reply:
I will bring this to Cyndi's attention when she comes back in case I missed something

Reply:
FWIW if we've tested this and it appears to resolve the issue then I think it's definitely preferable to the throttling code, so I'm fine with us merging. Didn't mean to hold up this PR with those concerns. Thanks for digging into this @lmarceau

Reply:
Cyndi is back on Monday, so I think it's fine if this isn't merged right away.